ocaml/testsuite/tests/formatting/test_locations.dlocations.ocamlc.reference

```diff
@@ -6,6 +6,8 @@
         pattern (test_locations.ml[17,534+8]..[17,534+11])
           Ppat_var "fib" (test_locations.ml[17,534+8]..[17,534+11])
         expression (test_locations.ml[17,534+14]..[19,572+34])
+          attribute "jane.erasable._builtin"
```

Comment:
I don't see this attribute listed in the BNF in `Jane_syntax`. I think it probably should, because it's part of the little grammar here. But upon closer inspection, I don't know how this gets here at all. The empty suffix seems to arise only from `Top_level`, but that seems never to happen (except by reading an existing `jane.erasable._builtin`). So something is off somewhere (quite possibly my understanding).

To be clear, I didn't review the code in Jane_syntax all that closely -- it will be short-lived and most mistakes would be easily caught during routine testing.

Reply:
This is the top-level attribute that introduces the Jane Syntax construct. (Same as, say, `jane.non_erasable.comprehension`.) My decision to not include it in the BNF mirrors Antal's decision to not include `jane.non_erasable.comprehension` in the `Comprehension` BNF.

This attribute is inserted by `Jane_syntax_parsing`, not by the syntactic arity&ndash;related code in `Jane_syntax.

Reply:
Ah -- that makes sense. Thanks.